includes/functions.php

```diff
@@ -34,12 +34,46 @@
  *
  * @param array $args Arguments for fetching stories.
  *
- * @return string|void
+ * @return void
  */
 function stories( $args = [] ) {
```

Comment:
We should have a less ambiguous/vague function name for this.

Maybe `display_stories()`? Or `render_stories()`?

cc @spacedmonkey for input

Reply:
How about the name `render_web_stories` as name. It is makes very clear what is does.

Reply:
Sorry, this is namespace right? So to use it, it would `\Google\Web_Stories\render_stories( [] ) `

Maybe `render_stories` stories. 

I think that render is cleaner than display, as sometimes, display_x means that version that is displayed and doesn't imply it is echoed.

Reply:
`render_stories()` works or me 👍